recipe_manager/models.py

```diff
@@ -0,0 +1,10 @@
+# models.py
+from django.db import models
+
+class Recipe(models.Model):
+    recipe_name = models.CharField(max_length=100)
+    instructions = models.TextField()
+
+class Ingredient(models.Model):
+    recipe = models.ForeignKey(Recipe, related_name='ingredients', on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
```

Comment:
I guess this works for this usecase, in a more realistic scenario it might probably be a many to many relationship... but for this task I think using a one to many is also good 😄

Reply:
the task was explicitly saying that I should "assume a given ingredient belongs only to one recipe, even if that means multiple Ingredient instances with the exact same name" hence the one to many:D

Reply:
btw task description here: https://www.notion.so/travelperk/Description-of-the-Django-Exercise-49da5411fb7149ed844d70baebf93efc

Reply:
Oh you're completely right, sorry about that then 🤣